doc/manual.md

```diff
@@ -6956,25 +6956,41 @@ All identifiers of a module are valid from the point of declaration until
 the end of the module. Identifiers from indirectly dependent modules are *not*
 available. The `system`:idx: module is automatically imported in every module.
 
-If a module imports an identifier by two different modules, each occurrence of
-the identifier has to be qualified unless it is an overloaded procedure or
-iterator in which case the overloading resolution takes place:
+If a module imports the same identifier from two different modules, the
+identifier must be disambiguated in one of the following ways:
```

Comment:
Good start, but the items 1, 2 are done by the compiler and 3 describes a way out for the programmer.

Reply:
I now (hopefully) phrased them all as ways out for the programmer, and moved qualification to the top